src/EventListener/SerializeListener.php

```diff
@@ -60,6 +60,9 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
         }
 
         $context = $this->serializerContextBuilder->createFromRequest($request, true, $attributes);
+        if ($includes = $request->attributes->get('_api_includes')) {
```

Comment:
Why include**s** here and include**d** the line below?

Reply:
I've fixed the inconsistency and went with `included` everywhere 👍